android/AndroidOpenGLESLessons/app/build.gradle

```diff
@@ -0,0 +1,31 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 23
+    buildToolsVersion "24.0.0"
+
+    defaultConfig {
+        applicationId "com.learnopengles.android"
+        minSdkVersion 9
```

Comment:
`GLES20.glVertexAttribPointer` (which I have replaced libgdx calls with) requires API 9 (was 8) so I have bumped `minSdkVersion` from 8 to 9, hopefully this is OK...